website/docs/community.mdx

```diff
@@ -0,0 +1,41 @@
+import Tabs from '@theme/Tabs'
```

Comment:
Not an issue but you don't need to import these if you're not using tabs in the page